app/controllers/address_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AddressController < ApplicationController
+  def create
+    user = User.find_by(verification_token: params[:verification_token])
```

Comment:
why is the user being found using verification_token?

Reply:
Will discuss this on wednesday